services/auth.py

```diff
@@ -67,3 +68,9 @@ def verify_access_token(token):
     if paylod and not paylod.get('grant_type') == 'refresh':
         return True, paylod
     return False
+
+
+def get_user_from_request(request):
+    token = request.headers['x-access-tokens']
+    paylod = jwt.decode(token, app.config['SECRET_KEY'], os.getenv('ALGORTHM'))
+    return User.query.filter_by(id=paylod.get('id')).first()
```

Comment:
why first? I think you expect one user with given id, so it should be `one` or `one_or_none`